src/main/asciidoc/java/http.adoc

```diff
@@ -23,6 +28,34 @@ HttpServerOptions options = new HttpServerOptions().setMaxWebsocketFrameSize(100
 HttpServer server = vertx.createHttpServer(options);
 ----
 
+=== Configuring an HTTP/2 server
+
+Vert.x supports HTTP/2 over TLS `h2` and over TCP `h2c`.
+
+To handle `h2` requests, TLS must be enabled with Application-Layer Protocol Negotiation:
+
+[source,java]
+----
+HttpServerOptions options = new HttpServerOptions()
+    .setUseAlpn(true)
+    .setSsl(true)
+    .setKeyStoreOptions(new JksOptions().setPath("/path/to/my/keystore"));
+
+HttpServer server = vertx.createHttpServer(options);
+----
+
+ALPN is a TLS extension that negotiates the protocol before the client and the server start to exchange data.
```

Comment:
Add link to https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation